analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ArrayPassedAsParams.cs

```diff
@@ -71,3 +71,25 @@ public class IndexerClass
 {
     public int this[params int[] i] => 1;
 }
+
+public class Repro6894
+{
+    //Reproducer for https://github.com/SonarSource/sonar-dotnet/issues/6894
+
+    public void Method(params object[] args) { }
+    public void MethodJuggedArray(params int[][] args) { }
+
+    public void CallMethod()
+    {
+        Method(new String[] { "1", "2" }); // Noncompliant, TP. Elements in args: {"1", "2"}
```

Comment:
```suggestion
        Method(new String[] { "1", "2" }); // Noncompliant, FP. Elements in args: {string[]}
```

Reply:
@martin-strecker-sonarsource  it's not the case :/ For me when I iterate over args it's giving me back two elements 1 and 2.
Could you maybe run a quick test on your side too?

Reply:
I added a comment there to clarify.